Code/src/main/java/nl/utwente/group10/ui/gestures/AbstractGesture.java

```diff
@@ -0,0 +1,14 @@
+package nl.utwente.group10.ui.gestures;
+
+import javafx.scene.Node;
+
+public abstract class AbstractGesture {
```

Comment:
Momenteel nergens, maar deze structuur is later waarschijnlijk bruikbaar voor geavanceerde gestures.